src/app/api/models/authorizationRuleNotFound.Error.ts

```diff
@@ -0,0 +1,14 @@
+/***********************************************************
```

Comment:
An interesting naming of the file. Is it intended since we have an error folder already?

Reply:
yeah -- this looks like a mistake.  will fix.